libraries/Microsoft.Bot.Builder.Azure.CosmosDb/CosmosDBKeyEscape.cs

```diff
@@ -7,7 +7,7 @@
 using System.Linq;
 using System.Text;
 
-namespace Microsoft.Bot.Builder.Azure
+namespace Microsoft.Bot.Builder.Azure.CosmosDb
```

Comment:
We should track the namespace change as a breaking change somewhere (let's chat in the standup where/how we want to do this)